static/app/views/alerts/wizard/options.tsx

```diff
@@ -205,47 +207,92 @@ const TRANSACTION_SUPPORTED_TAGS = [
 ];
 const SESSION_SUPPORTED_TAGS = [FieldKey.RELEASE];
 
+// This is purely for testing purposes, use with alert-allow-indexed feature flag
+const INDEXED_PERFORMANCE_ALERTS_OMITTED_TAGS = [
```

Comment:
Might not be perfect, based on this list:
https://docs.sentry.io/product/sentry-basics/search/searchable-properties/